articles/code_coverage.dd

```diff
@@ -34,8 +34,10 @@ $(LI It can be used to track down why a particular section of code
 
 $(LI Since execution counts are given for each line, it is possible
      to use the coverage analysis to reorder the basic blocks in
-     a function to minimize jmps in the most used path, thus
-     optimizing it. This is known as $(LINK2 https://en.wikipedia.org/wiki/Profile-guided_optimization, profile-guided optimization)
+     a function to minimize branches in the most used path, reducing stress
+     on the processor's pipeline (A run-of-the-mill X86 processor can usually handle 48 branches in its pipeline) - this technique
```

Comment:
Two things with this line. The "A" in the parentheses should be lowercase. And the dash before "this technique" should either be an em dash, a semicolon, or just a new sentence. I don't believe there's a macro for th HTML mdash entity, so a semi-colon or a new sentence would be easier.

Reply:
Now a new sentence.